static/src/javascripts/projects/commercial/modules/third-party-tags/imr-worldwide.ts

```diff
@@ -3,7 +3,7 @@ import { isInAuOrNz } from '../../../common/modules/commercial/geo-utils';
 
 // NOLCMB is a global function defined by the IMR worldwide library
 
-const guMetadata = {
+const guMetadata: Record<string, string> = {
```

Comment:
I would recommend using the `as const` assertion, here!

Reply:
I tried that originally but then it seemed like more effort than it was worth to then narrow `config.page.section`, a `string`, to `keyof typeof guMetadata` in order to use it to index into `guMetadata` (and I'm still not sure how to do this).